monster_ai.cpp

```diff
@@ -38,6 +38,9 @@
 #include "furniture_factory.h"
 #include "file_path.h"
 
+//Note below, a strong monster may be able to throw it's best item further than 10 squares, if it were more clever!
```

Comment:
There could be another interesting piece of work here. We could have thrown spears that the stronger monsters know they can throw 5 squares. etc.